diff -up openafs-1.8.2/src/afs/LINUX/osi_vfsops.c.5fix3 openafs-1.8.2/src/afs/LINUX/osi_vfsops.c
--- openafs-1.8.2/src/afs/LINUX/osi_vfsops.c.5fix3	2019-03-25 17:45:03.454406692 -0400
+++ openafs-1.8.2/src/afs/LINUX/osi_vfsops.c	2019-03-25 17:46:08.782578748 -0400
@@ -104,7 +104,11 @@ afs_fill_super(struct super_block *sb, v
    __module_get(THIS_MODULE);
 
     afs_globalVFS = sb;
+#if defined(SB_NOATIME)
+    sb->s_flags |= SB_NOATIME;
+#else
     sb->s_flags |= MS_NOATIME;
+#endif
     sb->s_blocksize = 1024;
     sb->s_blocksize_bits = 10;
     sb->s_magic = AFS_VFSMAGIC;
