diff -up openafs-1.6.20.2/src/afs/LINUX/osi_machdep.h.411fix openafs-1.6.20.2/src/afs/LINUX/osi_machdep.h
--- openafs-1.6.20.2/src/afs/LINUX/osi_machdep.h.411fix	2017-05-26 14:47:25.133547270 -0400
+++ openafs-1.6.20.2/src/afs/LINUX/osi_machdep.h	2017-05-26 14:48:09.229953218 -0400
@@ -145,10 +145,10 @@ static inline long copyinstr(char *from,
 #define NGROUPS NGROUPS_SMALL
 #endif
 
-#ifdef STRUCT_GROUP_INFO_HAS_GID
+
 /* compat macro for Linux 4.9 */
 #define GROUP_AT(gi,x)  ((gi)->gid[x])
-#endif
+
 
 typedef struct task_struct afs_proc_t;
 
