diff -up openafs-1.6.22.2/src/rx/rx_packet.c.auristorfix openafs-1.6.22.2/src/rx/rx_packet.c
--- openafs-1.6.22.2/src/rx/rx_packet.c.auristorfix	2018-08-15 11:12:37.543431500 -0400
+++ openafs-1.6.22.2/src/rx/rx_packet.c	2018-08-15 11:15:25.479912764 -0400
@@ -1654,6 +1654,7 @@ rxi_SplitJumboPacket(struct rx_packet *p
     np->header = p->header;
     np->header.serial = p->header.serial + 1;
     np->header.seq = p->header.seq + 1;
+    np->header.userStatus = 0;
     np->header.flags = jp->flags;
     np->header.spare = jp->cksum;
 
@@ -2689,6 +2690,7 @@ rxi_SendSpecial(struct rx_call *call,
     p->header.seq = 0;
     p->header.epoch = conn->epoch;
     p->header.type = type;
+    p->header.userStatus = 0;
     p->header.flags = 0;
     if (conn->type == RX_CLIENT_CONNECTION)
 	p->header.flags |= RX_CLIENT_INITIATED;
@@ -2811,6 +2813,7 @@ rxi_PrepareSendPacket(struct rx_call *ca
     p->header.seq = seq;
     p->header.epoch = conn->epoch;
     p->header.type = RX_PACKET_TYPE_DATA;
+    p->header.userStatus = 0;
     p->header.flags = 0;
     p->header.spare = 0;
     if (conn->type == RX_CLIENT_CONNECTION)
