diff -up openafs-1.6.20.1/src/packaging/RedHat/openafs-client.service.fhs openafs-1.6.20.1/src/packaging/RedHat/openafs-client.service
--- openafs-1.6.20.1/src/packaging/RedHat/openafs-client.service.fhs	2016-12-29 19:15:18.484330028 -0500
+++ openafs-1.6.20.1/src/packaging/RedHat/openafs-client.service	2016-12-29 19:16:49.851140901 -0500
@@ -9,9 +9,9 @@ Type=forking
 RemainAfterExit=true
 EnvironmentFile=/etc/sysconfig/openafs
 ExecStartPre=/sbin/modprobe openafs
-ExecStart=/usr/vice/etc/afsd $AFSD_ARGS
+ExecStart=/usr/sbin/afsd $AFSD_ARGS
 ExecStop=/bin/umount /afs
-ExecStop=/usr/vice/etc/afsd -shutdown
+ExecStop=/usr/sbin/afsd -shutdown
 ExecStop=/sbin/rmmod libafs
 KillMode=none
 
diff -up openafs-1.6.20.1/src/packaging/RedHat/openafs-server.service.fhs openafs-1.6.20.1/src/packaging/RedHat/openafs-server.service
--- openafs-1.6.20.1/src/packaging/RedHat/openafs-server.service.fhs	2016-12-29 19:15:26.931220093 -0500
+++ openafs-1.6.20.1/src/packaging/RedHat/openafs-server.service	2016-12-29 19:17:20.310744474 -0500
@@ -4,7 +4,7 @@ After=syslog.target network.target
 
 [Service]
 EnvironmentFile=-/etc/sysconfig/openafs
-ExecStart=/usr/afs/bin/bosserver -nofork $BOSSERVER_ARGS
+ExecStart=/usr/sbin/bosserver -nofork $BOSSERVER_ARGS
 ExecStop=/usr/bin/bos shutdown localhost -wait -localauth
 
 [Install]
