diff -up src/rxkad/Makefile.in.fpic1 src/rxkad/Makefile.in
--- src/rxkad/Makefile.in.fpic1	2015-02-10 15:02:08.895594119 -0500
+++ src/rxkad/Makefile.in	2015-02-10 15:03:03.880584970 -0500
@@ -90,10 +90,10 @@ ticket5_keytab.o: rxkad.h
 ticket.o: rxkad.h
 
 ticket.o: ticket.c lifetimes.h ${INCLS}
-	$(CC) $(CFLAGS) -c ${srcdir}/ticket.c
+	$(CCOBJ) $(CFLAGS) -c ${srcdir}/ticket.c
 
 ticket5.o: ticket5.c v5gen.c v5der.c v5gen-rewrite.h ${INCLS}
-	$(CC) -I${srcdir} $(CFLAGS) -c ${srcdir}/ticket5.c
+	$(CCOBJ) -I${srcdir} $(CFLAGS) -c ${srcdir}/ticket5.c
 
 crc.o: crc.c ${INCLS}
 
