diff -up openafs-1.6.20.1/src/packaging/RedHat/openafs-client.service.envvars openafs-1.6.20.1/src/packaging/RedHat/openafs-client.service
--- openafs-1.6.20.1/src/packaging/RedHat/openafs-client.service.envvars	2016-12-29 19:19:52.111768803 -0500
+++ openafs-1.6.20.1/src/packaging/RedHat/openafs-client.service	2016-12-29 19:20:30.214272904 -0500
@@ -9,7 +9,7 @@ Type=forking
 RemainAfterExit=true
 EnvironmentFile=/etc/sysconfig/openafs
 ExecStartPre=/sbin/modprobe openafs
-ExecStart=/usr/sbin/afsd $AFSD_ARGS
+ExecStart=/usr/sbin/afsd $AFSD_ARGS $CACHE $OPTIONS $CONFDIR
 ExecStop=/bin/umount /afs
 ExecStop=/usr/sbin/afsd -shutdown
 ExecStop=/sbin/rmmod libafs
