diff -up openafs-1.6.20.2/src/packaging/RedHat/openafs-client.service.cellservdbfix openafs-1.6.20.2/src/packaging/RedHat/openafs-client.service
--- openafs-1.6.20.2/src/packaging/RedHat/openafs-client.service.cellservdbfix	2017-05-25 15:53:10.126072131 -0400
+++ openafs-1.6.20.2/src/packaging/RedHat/openafs-client.service	2017-05-25 15:54:08.791317724 -0400
@@ -9,8 +9,6 @@ Type=forking
 RemainAfterExit=true
 EnvironmentFile=/etc/sysconfig/openafs
 ExecStartPre=/bin/bash -c "fs sysname > /dev/null 2>/dev/null; test $? -ne 0 || (echo AFS client appears to be running -- not starting && exit 1)"
-ExecStartPre=/bin/sed -n 'w/usr/vice/etc/CellServDB' /usr/vice/etc/CellServDB.local /usr/vice/etc/CellServDB.dist
-ExecStartPre=/bin/chmod 0644 /usr/vice/etc/CellServDB
 ExecStartPre=/sbin/modprobe openafs
 ExecStart=/usr/vice/etc/afsd $AFSD_ARGS
 ExecStop=/bin/umount /afs
