diff -up openafs-1.8.2/src/afs/LINUX/osi_file.c.5fix2 openafs-1.8.2/src/afs/LINUX/osi_file.c
--- openafs-1.8.2/src/afs/LINUX/osi_file.c.5fix2	2019-03-25 17:30:19.307065706 -0400
+++ openafs-1.8.2/src/afs/LINUX/osi_file.c	2019-03-25 17:33:18.052539835 -0400
@@ -23,11 +23,17 @@
 #include "osi_compat.h"
 
 #ifndef CURRENT_TIME
-# ifdef IATTR_TAKES_64BIT_TIME
-#  define CURRENT_TIME		(current_kernel_time64())
+# if defined(HAVE_LINUX_KTIME_GET_COARSE_REAL_TS64)
+#   define AFS_CURRENT_TIME(x) (ktime_get_coarse_real_ts64((x)))
 # else
-#  define CURRENT_TIME            (current_kernel_time())
+#  ifdef IATTR_TAKES_64BIT_TIME
+#    define AFS_CURRENT_TIME(x) do {*(x) = current_kernel_time64();} while (0)
+#  else
+#    define AFS_CURRENT_TIME(x) do {*(x) = current_kernel_time();} while (0)
+#  endif
 # endif
+#else
+# define AFS_CURRENT_TIME(x) do {*(x) = CURRENT_TIME;} while(0)
 #endif
 
 int cache_fh_type = -1;
@@ -200,7 +206,7 @@ osi_UFSTruncate(struct osi_file *afile,
 #endif
     newattrs.ia_size = asize;
     newattrs.ia_valid = ATTR_SIZE | ATTR_CTIME;
-    newattrs.ia_ctime = CURRENT_TIME;
+    AFS_CURRENT_TIME(&newattrs.ia_ctime);
 
     /* avoid notify_change() since it wants to update dentry->d_parent */
 #ifdef HAVE_LINUX_SETATTR_PREPARE
diff -up openafs-1.8.2/src/cf/linux-test4.m4.5fix2 openafs-1.8.2/src/cf/linux-test4.m4
--- openafs-1.8.2/src/cf/linux-test4.m4.5fix2	2019-03-25 17:35:06.635827856 -0400
+++ openafs-1.8.2/src/cf/linux-test4.m4	2019-03-25 17:36:55.095115541 -0400
@@ -189,7 +189,12 @@ AC_DEFUN([LINUX_IATTR_64BIT_TIME], [
                        [#include <linux/fs.h>
                         #include <linux/timekeeping.h>],
                        [struct iattr _attrs;
-                        _attrs.ia_ctime = current_kernel_time64();],
+                       #if defined(HAVE_LINUX_KTIME_GET_COARSE_REAL_TS64)
+                        ktime_get_coarse_real_ts64(&_attrs.ia_ctime);
+                       #else
+                        _attrs.ia_ctime = current_kernel_time64();
+                       #endif
+                       ],
                        [IATTR_TAKES_64BIT_TIME],
                        [define if struct iattr->ia_ctime takes struct timespec64],
                        [])
